src/pint/toa.py

```diff
@@ -77,10 +77,10 @@
 def get_TOAs(
     timfile,
     ephem=None,
-    include_bipm=True,
-    bipm_version=bipm_default,
-    include_gps=True,
-    planets=False,
+    include_bipm=None,
```

Comment:
What is the reasoning for removing the default values from the signature and putting them into the code?  Seems like it makes the signature harder to understand.  Is this a recommended practice?

Reply:
This makes it possible to load a pickle of unknown settings (as in one test case). It is custom to use "None" to indicate an unspecified default, and to put the default in the docstring, if the code wants to distinguish between unspecified and an explicit value.